pkg/windows/buildenv/salt-call.bat

```diff
@@ -2,10 +2,18 @@
 :: Script for starting the Salt-Minion
 :: Accepts all parameters that Salt-Minion Accepts
 
+:: Get current codepage, change codpage to Unicode
+@for /f "tokens=2 delims=:." %%x in ('chcp') do set cp=%%x
+@chcp 65001>nul
```

Comment:
This is a different style than other files within salt. I think no space before the > makes it a bit harder to read, but that is subjective. Looking at other files, these are the other styles I have observed:

```
@chcp 65001 >nul
@chcp 65001 1>nul
@chcp 65001 >nul 2>&1
```

Of course, the last one also swallows up stderr, I am not sure you want that. It would be nice to be consistent throughout. This comment is only a style comment, and hence does not gate this PR.